src/py-opentimelineio/opentimelineio/core/_core_utils.py

```diff
@@ -228,6 +228,11 @@ def __deepcopy__(self, memo):
                     and name not in klass.__abstractmethods__
             ):
                 setattr(mapClass, name, _im_func(func))
+                if name.startswith('__') or name.endswith('__'):
+                        continue
+
+                # Hide the method frm Sphinx doc. See https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists
+                getattr(mapClass, name).__doc__ += '\n\n:meta private:'
```

Comment:
Little trick to unclutter the reference documentation with unnecessary (well at least IMO) methods.